packages/@aws-cdk/core/lib/cfn-parse.ts

```diff
@@ -591,6 +598,39 @@ export class CfnParser {
     }
   }
 
+  private handleRulesIntrinsic(key: string, object: any): any {
+    // Rules have their own set of intrinsics:
+    // https://docs.aws.amazon.com/servicecatalog/latest/adminguide/reference-template_constraint_rules.html
```

Comment:
This doc actually describes each of the rules; might be a better landing point:
```suggestion
    // https://docs.aws.amazon.com/servicecatalog/latest/adminguide/intrinsic-function-reference-rules.html
```

Reply:
Sure. Done.